.gitignore

```diff
@@ -4,3 +4,4 @@
 composer.lock
 node_modules/
 Modules/
+/.project
```

Comment:
This should be in your global `.gitignore`. IDE/Text Editor config files are typically per-developer (since not everyone uses the same one - unless your workplace forces you to use a standard editor), so it should be the responsibility of the developer to make sure their individual config files are ignored and not rely on the project's `.gitignore` to do so.